openreview/journal/invitation.py

```diff
@@ -492,6 +520,7 @@ def set_reviewer_assignment_acknowledgement_invitation(self):
                     'signatures': { 'const': [editors_in_chief_id] },
                     'maxReplies': { 'const': 1 },
                     'duedate': { 'const': '${params.duedate}' },
+                    # 'expdate': { 'const': None },
```

Comment:
currently this line will cause invitation validation to fail
discussed with @carlosmondra and he will make the changes to allow const null

when the changes are added, unexpire invitation can be removed.